superset/models/helpers.py

```diff
@@ -66,14 +63,6 @@ class ImportMixin:
     # The names of the attributes
     # that are available for import and export
 
-    @classmethod
```

Comment:
@willbarrett I am really interested in your opinion about this change. I assume this method is private because:
- it might change in future, or
- it's supposed to be used only in `ImportMixin` class.
But since this method is used on other instances (not only `cls`) I suggested making it public. WDYT?

Reply:
Yes, I think making it public is a good idea.